### DISCUSSION TITLE
Comments: My best month in years

### DISCUSSION THREAD

**Discussion by qwertie** [qwertie/ecsharp#130] [Web site comments]
For http://loyc.net/2021/best-month-in-years.html

**Comment by dadhi** [comment]
@qwertie Happy NY, and I hope you will be able to promote your ideas and get some positive reaction for whatever you're doing! And the best luck with the other life things.

Looking forward for the new LEMP release and for the videos.

**Comment by dzmitry-lahoda** [comment]
i think you would be more happy, valuable and just richer by applying your skills and knowledge to make for of C# champions features get into 10 :), contribute to roslyn i mean.

**Reply by qwertie** [reply]
The thing about that is, in the long run I really want to shift from C#-related things (especially if that means giving my free time to MS for MS's benefit - I am perhaps a little sore about how my past 'interactions' with MS people never went anywhere). The original goals of Loyc - code conversion and interoperability - remain on my agenda, and I'd like to somehow promote the use of [Loyc trees](http://loyc.net/loyc-trees/) in new and existing languages, and I'd like to design some kind of extensible language myself or collaborate with others on an existing project if there is a suitably ambitious one out there.

If contributing to Roslyn could somehow lead to success on other fronts, I might be interested.

**Reply by qwertie** [reply]
Also it should be noted that I'm fairly sure C# will never support certain central features of Enhanced C# (like `matchCode` and `define`) no matter what I do.

**Comment by kfawell** [comment]
Hello. I have been using MMap. Works so well. Considerably faster than other immutable or persistent maps. Thank you very much.
